lib/openscap_parser/rule.rb

```diff
@@ -16,19 +16,19 @@ def severity
     end
 
     def title
-      @title ||= @rule_xml.at_css('title').children.first.text
```

Comment:
I noticed this `.children.first.text` returns the same thing as `.text`, so this _should_ be just a refactor

Reply:
smart proxy is running on older ruby versions that do not yet support the `.&` syntax, so we need to avoid that

Reply:
Good to know! I'll update it.